### PR TITLE
379 - configuration fix

### PR DESCRIPTION
**Pull request by dorians**

### CODE REVIEW

File: src/main/scala/tech/cryptonomic/conseil/tezos/TezosPlatformDiscoveryOperations.scala [hunk 1]
```diff
@@ -36,6 +36,7 @@ object TezosPlatformDiscoveryOperations {
       case "numeric" => DataType.Decimal
       case "bool" => DataType.Boolean
       case "hash" => DataType.Hash
+      case "accountAddress" => DataType.AccountAddress
```
Comment (ivanopagano) [inline src/main/scala/tech/cryptonomic/conseil/tezos/TezosPlatformDiscoveryOperations.scala]: Do we really want this to be case-sensitive in the end?
@dorians @anonymoussprocket ?

Comment (anonymoussprocket) [reply src/main/scala/tech/cryptonomic/conseil/tezos/TezosPlatformDiscoveryOperations.scala]: it just following the other values, Boolean, Decimal, etc. I figure camel case is appropriate for a two-word value. I'm open to suggestion however depending on what common enum naming practices are in scala.

Comment (ivanopagano) [reply src/main/scala/tech/cryptonomic/conseil/tezos/TezosPlatformDiscoveryOperations.scala]: If we want to make it consistent with typenames casing in scala, we should go CamelCase.

But I wonder if we might want to follow the so-called [Postel's Law](https://en.wikipedia.org/wiki/Robustness_principle) about being more robust to accept values

Comment (dorians) [reply src/main/scala/tech/cryptonomic/conseil/tezos/TezosPlatformDiscoveryOperations.scala]: I would opt for using a different field. This method `maps type from DB to type used in query` which might be confusing.

Comment (ivanopagano) [reply src/main/scala/tech/cryptonomic/conseil/tezos/TezosPlatformDiscoveryOperations.scala]: I don't get it, isn't this what we're trying to do? Map query types, with aliasing, to actual database types?
Is this used for a different sort of operation?

Comment (dorians) [reply src/main/scala/tech/cryptonomic/conseil/tezos/TezosPlatformDiscoveryOperations.scala]: It's used for metadata overrides and it's not related with database types directly.

**Pull request merged**